docs/docs/tutorial.md

```diff
@@ -549,7 +549,7 @@ var CommentForm = React.createClass({
     var author = this.state.author.trim();
     var text = this.state.text.trim();
     if (!text || !author) {
-      return;
+      return false;
```

Comment:
Was using 15.0.1 but switching to the newest version fixed the problem. The zip included with the tutorial uses 15.0.1, which may be the cause of the issue.